proc_scripts/load_data.py

```diff
@@ -137,10 +139,10 @@ def proc_square_wave_capture(s):
 
 postproc_dict = {'ag_IR2H':proc_bruker_deut_IR_withecho_mancyc,
         'ab_ir2h':proc_bruker_deut_IR_mancyc,
-        'CPMG':proc_CPMG,
-        'Hahn_echoph':proc_Hahn_echoph,
-        'spincore_nutation':proc_nutation,
-        'spincore_IR':proc_spincore_IR,
+        'spincore_CPMG_v1':proc_CPMG,
+        'spincore_Hahn_echoph_v1':proc_Hahn_echoph,
+        'spincore_nutation_v1':proc_nutation,
+        'spincore_IR_v1':proc_spincore_IR,
         'spincore_ODNP_v1':proc_spincore_ODNP_v1,
-        'square_wave_capture':proc_square_wave_capture}
+        'square_wave_capture_v1':proc_square_wave_capture}
```

Comment:
see my earlier comment about being sufficiently unique

Reply:
```
johnf@DESKTOP-O82GMEV MINGW64 ~/git_repos/proc_scripts (alec_fitdata_rebased)
$ grep . -rie "'CPMG'"
Binary file ./.proc_CPMG.py.swp matches
./ipynb/20190509_homogeneous_Liouville_simulation.py:    fl.next('cpmg')
./proc_CPMG.py:        #('200221_CPMG_TEMPOLgel_3p0_1','test_equip','signal','CPMG','deadtime=5'),
./proc_CPMG.py:        #('200304_CPMG_2p6_1','test_equip','signal','CPMG','deadtime=5'),
./proc_CPMG.py:        #('200305_CPMG_3p7_2','test_equip','signal','CPMG','deadtime=5'),
(base)
```

Reply:
again -- see the earlier comment about being sufficiently unique.  Here, you just rolled back my changes to get the code to work

Reply:
to clarify my previous comments, when you run `grep . -rie "'CPMG'"` from the shell, you shouldn't see anything

Reply:
(this is related to the [previous comment](https://github.com/jmfrancklab/proc_scripts/pull/14#discussion_r442937681))